admin/database/database.go

```diff
@@ -264,6 +269,7 @@ type UpdateProjectOptions struct {
 	ProdVariables        map[string]string
 	ProdDeploymentID     *string
 	ProdSlots            int
+	ProdTTL              *int64
```

Comment:
Make sure every place in the code (except tests) that uses `UpdateProjectOptions` (search for it in editor) passes a value for `ProdTTL`